.gitignore

```diff
@@ -28,3 +29,6 @@ CMakeLists.txt
 
 # Atom files
 .tags
+
+# Bazel
+bazel-*/
```

Comment:
This doesn't ignore the symlinks created by bazel in my Linux, maybe it should be `bazel-*` only (with no `/`)